### PR TITLE
Add MSP Commands module

### PR DESCRIPTION
**Pull request by kiequoo**

### CODE REVIEW

File: msgs/src/main/scala/com/thenewmotion/ocpi/msgs/v2_1/Commands.scala [hunk 1]
```diff
@@ -1,48 +1,136 @@
...
+    val values = Iterable(ReserveNow, StartSession, StopSession, UnlockConnector)
+  }
+
+  abstract class Command(val name: CommandName) {
+    def responseUrl: Url
+  }
+
+  private[ocpi] def callbackUrl(
+    baseUrl: Url,
+    cmdName: CommandName,
```
Comment (DStranger) [inline msgs/src/main/scala/com/thenewmotion/ocpi/msgs/v2_1/Commands.scala]: Why do you need a command name here? Uuid should be enough.

Comment (kiequoo) [reply msgs/src/main/scala/com/thenewmotion/ocpi/msgs/v2_1/Commands.scala]: True, the examples given in the OCPI docs had both the command name and an identifier so I went with that....

File: endpoints-msp-commands/src/main/scala/com/thenewmotion/ocpi/commands/CommandResponseRoute.scala [hunk 2]
```diff
@@ -24,15 +24,11 @@ class CommandResponseRoute(
   private val CommandNameSegment = Segment.flatMap(x => CommandName.values.find(_.name == x))
 
   private[commands] def routeWithoutRh(apiUser: GlobalPartyId)(implicit executionContext: ExecutionContext) =
-    authPathPrefixGlobalPartyIdEquality(apiUser) {
```
Comment (DStranger) [inline endpoints-msp-commands/src/main/scala/com/thenewmotion/ocpi/commands/CommandResponseRoute.scala]: Why remove this line?

Comment (kiequoo) [reply endpoints-msp-commands/src/main/scala/com/thenewmotion/ocpi/commands/CommandResponseRoute.scala]: There is no need to include the global party id in the URL?  Was there due to copy/paste :)

Comment (DStranger) [reply endpoints-msp-commands/src/main/scala/com/thenewmotion/ocpi/commands/CommandResponseRoute.scala]: Oh, true...

File: msgs/src/main/scala/com/thenewmotion/ocpi/msgs/v2_1/Commands.scala (hunk 1 above)
Comment (DStranger) [reply msgs/src/main/scala/com/thenewmotion/ocpi/msgs/v2_1/Commands.scala]: That's it then, other that this.

**Pull request merged**